signalfx/util.go

```diff
@@ -208,7 +208,7 @@ func getColorScaleOptions(d *schema.ResourceData) []*chart.SecondaryVisualizatio
 }
 
 func getValueUsingMaxFloatAsDefault(v float64) *float32 {
-	if v >= math.MaxFloat32 || v <= -math.MaxFloat32 {
```

Comment:
Also is there any good reason not to just clamp the value to max/min float32 if it is OOB?